connect/runtime/src/main/java/org/apache/kafka/connect/cli/ConnectStandalone.java

```diff
@@ -37,98 +34,65 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.net.URI;
-import java.util.Arrays;
-import java.util.Collections;
 import java.util.Map;
 
 /**
  * <p>
- * Command line utility that runs Kafka Connect as a standalone process. In this mode, work is not
- * distributed. Instead, all the normal Connect machinery works within a single process. This is
- * useful for ad hoc, small, or experimental jobs.
- * </p>
- * <p>
- * By default, no job configs or offset data is persistent. You can make jobs persistent and
- * fault tolerant by overriding the settings to use file storage for both.
+ * Command line utility that runs Kafka Connect as a standalone process. In this mode, work (connectors and tasks) is not
+ * distributed. Instead, all the normal Connect machinery works within a single process. This is useful for for development
+ * and testing Kafka Connect on a local machine.
```

Comment:
I've added it back 👍 

Off topic, but I just realized that the standalone mode interestingly stores configs in-memory but offsets in a file by default. By any chance, are you aware of why this is the case - i.e. why aren't configs made persistent via file storage too?

Reply:
I don't know the history here, but if I had to guess, two possibilities come to mind (which are not mutually exclusive):
- Offsets are persisted for source connectors in order to provide symmetry between source and sink (since sink connector offsets are stored by default as consumer offsets in both standalone and distributed mode, which causes them to persist beyond the lifetime of any single worker)
- Standalone mode may originally not have supported the REST API, in which case, the assumption would be that the configs are already stored on disk somewhere in order to be passed to the worker on startup as filenames

Reply:
Thanks, that makes a lot of sense. Just curious, would adding a `FileConfigBackingStore` to the standalone mode require a KIP since we would be adding a new worker configuration for the file name (although we could default to falling back on the MemoryConfigBackingStore)? Or can we get away without one because it's the standalone mode which isn't used for production systems?

Reply:
We'd definitely need a KIP for that; standalone mode is still part of the public interface for the project.

I'm also not sure about the assertion that it's not used for production systems--it may not be supported for production use by companies that offer support or other services for Kafka, but AFAIK Apache doesn't have an official stance one way or the other beyond the recommendations we lay out in our docs, none of which go so far as to state that standalone mode is not intended for production use.